developer/classes_overview/comp_map_relocation_manager.md

```diff
@@ -20,10 +20,10 @@ public void Start{
     relocationManager.onMapDownloadStarted.AddListener(OnMapStartDownloading);
 
     //Called when the position in map has been found after relocation request
-    relocationManager.onPositionFound.AddListener(OnPositionMatched);
+    relocationManager.onLocationFound.AddListener(OnPositionMatched);
```

Comment:
Shouldn't we keep `onPositionFound()` since this ones returns the matched position?

Reply:
Changes have been made on latest patch to with LocationFound and LocationNotFound because positionfound returns the position datas and can't be use in inspector